### PR TITLE
Raidboss: Ruby Weapon EX

### PR DESCRIPTION
**Pull request by VanSlanzar**
Translation fr missing

### CODE REVIEW

**Comment by VanSlanzar** [comment]
check error, if i must to changed one line say me. sorry, i'm beginning.

File: ui/raidboss/data/05-shb/trial/ruby-weapon-ex.js [hunk 1]
```diff
@@ -343,11 +347,13 @@
         if (matches.y < 90) {
           return {
             en: 'Comets N/S',
+            fr: 'Comètes N/S',
             ko: '남/북 운석 낙하',
           };
         } else if (matches.x < 90) {
           return {
             en: 'Comets E/W',
+            fr: 'Comètes E/O',00
```
Comment (quisquous) [inline ui/raidboss/data/05-shb/trial/ruby-weapon-ex.js]: It looks like this `00` here is the problem, based on the error messages from Travis.

**Comment by quisquous** [comment]
I'm happy to help.  Take a look at https://github.com/quisquous/cactbot/blob/master/CONTRIBUTING.md#build-problems and see if that helps any to track these errors down.

File: ui/raidboss/data/05-shb/trial/ruby-weapon-ex.js (hunk 1 above)
Comment (VanSlanzar) [reply ui/raidboss/data/05-shb/trial/ruby-weapon-ex.js]: oo, ty my son have touch at the moment my macro. sorry really.

Comment (quisquous) [reply ui/raidboss/data/05-shb/trial/ruby-weapon-ex.js]: No apologies needed! :)

**Pull request merged**